### ISSUE TITLE
Erroneous inconsistent indentation calls

### ISSUE THREAD

**Issue by evanworley** [clutchski/coffeelint#156]
The inconsistent indentation rule is firing for some code that is consistent. For example

```
callback(value)
  .done((rs) ->
    $input.remove()
  ).fail((error) ->
    $input.remove() # The error is reported on this line.
  )
```
